data/themes/default/style.css

```diff
@@ -858,7 +858,7 @@ lmms--gui--OrganicInstrumentView lmms--gui--Knob#volKnob {
 }
 
 lmms--gui--Sf2InstrumentView lmms--gui--Knob {
-	color: #ff00ea;
+	color: #00ff6e;
```

Comment:
What are these changes for?

Reply:
To get green colour on the knob. I have edited some pngs too to get green colour on areas where there was pink before

Reply:
Aight!